lang/cs/Org.Apache.REEF.IO.Tests/TestLocalFileSystem.cs

```diff
@@ -157,6 +157,27 @@ public void TestGetChildren()
             fs.DeleteDirectory(directoryUri);
         }
 
+        [Fact]
+        public void TestDeleteDirectory()
+        {
+            var fs = GetFileSystem();
+            var directoryUri = new Uri(Path.Combine(Path.GetTempPath(), TempFileName) + "/");
+            string childDirName = "childDir";
+            const string fileName = "testfile";
+            fs.CreateDirectory(directoryUri);
+            var childDirUri = new Uri(directoryUri, childDirName);
+            fs.CreateDirectory(childDirUri);
+            var fileUri = new Uri(directoryUri, fileName);
+            MakeRemoteTestFile(fs, fileUri);
+            var childDirFileUri = new Uri(directoryUri, $"{childDirName}/{fileName}");
+            MakeRemoteTestFile(fs, childDirFileUri);
+            fs.DeleteDirectory(directoryUri);
```

Comment:
This block of code is very dense. Please reorg to make more readable. 

i.e.

//Create directory

//Create sub file

//Create sub directory

Reply:
fixed

---
In reply to: [194553594](https://github.com/apache/reef/pull/1474#discussion_r194553594) [](ancestors = 194553594)